### DISCUSSION TITLE
Can't find sensors for sun, brightness and temperature

### DISCUSSION THREAD

**Discussion by frank2604** [basnijholt/adaptive-lighting#819] [Q&A]
Hi, I want to add some graphs to my dashboard that shows the graphs like at the end of your documentation, but I can't find the sensors for that. The integration has only the 4 switch entities.

Thanks

**Comment by Djelibeybi** [comment]
You need to create helper entities that extract those values from the attributes of the Adaptive Lighting switch. See #890 for examples of what the templates would need to look like.
